src/modules/universe/RadixUniverse.ts

```diff
@@ -165,15 +154,15 @@ export default class RadixUniverse {
         return new Promise<RadixNodeConnection>((resolve, reject) => {
             // Find active connection, return
             for (const node of this.connectedNodes) {
-                if (node.isReady() && this.canNodeServiceShard(node.node, shard)) {
+                if (node.isReady() && node.node.canServiceShard(shard)) {
```

Comment:
Can a `node` have a `node`? Or is it just unfortunate naming of two different types?

Reply:
The first one is a nodeConnection really, I should rename.